cdap-docs/deploy.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+
+# Copyright © 2015 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+################################################################################
+### Deploy script for docs
+# Deploys zip files created by build scripts
+# First parameter is cdap (default)
+# Second parameter is version
+# third parameter is the remote directory to be created on the web/doc server
+################################################################################
+DEBUG=${DEBUG:-no}
+PROJECT=${1:-cdap}
+PROJECT_DOCS=${PROJECT}-docs
+#PROJECT_PATH=
+VERSION=${2:-2.8.1}
```

Comment:
does it make sense to have defaults?  Will the default ever be useful?  If I understand correctly, the build should always provide an argument, and so this script should probably just validate and fail if no args given.